src/lib/cloudfront.ts

```diff
@@ -0,0 +1,45 @@
+import * as AWS from 'aws-sdk';
+import * as bluebird from 'bluebird';
+import { ServiceResponse, ServiceError } from './service';
+
+export async function flushDistribution(
+  distributionID: string,
+  awsAccessKeyID: string,
+  awsSecretAccessKey: string
+): bluebird<ServiceResponse> {
+  return flushURLs(distributionID, ['/'], awsAccessKeyID, awsSecretAccessKey);
+}
+
+export async function flushURLs(
+  distributionID: string,
+  urls: string[],
+  awsAccessKeyID: string,
+  awsSecretAccessKey: string
+): bluebird<ServiceResponse> {
```

Comment:
Not sure to understand why native promise are not used here.

Reply:
For information, bluebird and `async` keyword are incompatible in ES6. Typescript throw an error. For the moment, I let the target to ES5.